src/app/shared/issue/label/label.component.css

```diff
@@ -22,8 +22,8 @@
 }
 
 .labelInfo {
-  display: grid;
-  grid-template-columns: 20% auto;
+  display: flex;
```

Comment:
These changes seem irrelevant to the issue of text size. Is there a reason for changing from grid to flex?

Reply:
For reasons unknown to me, when on safari, if it remains as grid, the position of the label and the colored box is off:

<img width="228" alt="image" src="https://user-images.githubusercontent.com/67156011/179157780-ebe5c89c-1d1d-4e92-86ab-fdbea0732e20.png">

hence i removed grid and used flex instead to fix this issue

Reply:
Interesting that I do not have the same issue 
![image](https://user-images.githubusercontent.com/9072299/179379498-3a76ffd6-7a0d-446c-90fc-434739331476.png)
What is your screen resolution?

Reply:
Try changing the grid template to a fixed size, ie `20em auto 20em`, since we cannot enforce column widths with flex without adding more complicated css